src/System.Net.Http/src/netcore50/System/Net/HttpClientHandler.cs

```diff
@@ -321,7 +321,10 @@ public long MaxRequestContentBufferSize
 
         public int MaxResponseHeadersLength
         {
-            get { return 0; } // TODO: Issue #8541 - return default value used in Wininet.
+            // Windows.Web.Http is built on WinINet. There is no maximum limit (except for out of memory)
+            // for received response headers. So, returning 0 (indicating no limit) is appropriate.
+            get { return 0; }
```

Comment:
0 is deemed an invalid value in the other implementations, e.g.
https://github.com/dotnet/corefx/blob/5dcc7a499fc61f0b67456f638161af3cb12a3081/src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs#L428-L434
Should those be updated to treat 0 as meaning no limit?